### DISCUSSION TITLE
Add a divider item

### DISCUSSION THREAD

**Discussion by calypso15** [elchininet/custom-sidebar#214] [Ideas]
The basic idea is just a non-interactable horizontal line to help visually group items. Collapsible groups would be perhaps a better, but more labor intensive option.

**Comment by elchininet** [comment]
Hi @calypso15,
That would not be an easy feature to implement, in part [because of this](https://github.com/elchininet/custom-sidebar/discussions/163#discussioncomment-11120989) but also because right now `order` option expects to have only items related to a sidebar anchor and there are a lot of code that depend on this. But it is something good to take into account during a refactoring of the code.

**Comment by elchininet** [comment]
Hi @calypso15,
Check [the latest release](https://github.com/elchininet/custom-sidebar/releases/tag/v8.5.0).

**Reply by calypso15** [reply]
Oh that's awesome! You da man!
